rosette/rbl/rosette/space.rbl

```diff
@@ -111,15 +111,17 @@
     [[ctxt [code env]] ctxt-tuple]
     [[channels unification_ptrns product_ptrns persistent] location]
         ;;; the channel and the pattern of incoming messages to look for are destructured and bound
-    [_ (seq
-        (print (->symbol "DEBUG: Calling consume("))
-        (print channels)
-        (print (->symbol ", "))
-        (print unification_ptrns)
-        (print (->symbol ", "))
-        (print product_ptrns)
-        (print (->symbol ")"))
-        (p "")
+    [_  (if flag-verbose
+            (seq
+                (print (->symbol "DEBUG: Calling consume("))
+                (print channels)
+                (print (->symbol ", "))
+                (print unification_ptrns)
+                (print (->symbol ", "))
+                (print product_ptrns)
+                (print (->symbol ")"))
+                (p "")
+            )
         )]
```

Comment:
In lisp, you put the closing parents on the last line. No hanging parens.

Reply:
So, this might look like the following:

```
[_ (if flag-verbose
        (seq (print (->symbol "DEBUG: Calling consume("))
             (print channels)
             (print (->symbol ","))
             ...
             (p "")))]
```

Reply:
Done